### ISSUE TITLE
Too many boundary boxes

### ISSUE THREAD

**Issue by theman162** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#485]
After finishing training the model,once i runned using the webcam.py file there are many boundary boxes appearing and detecting the object.I would like to know the reason for this issue and solutions

**Comment by suyashhchougule** [comment]
try using NMS suppression

**Comment by prayagpawar** [comment]
Do u want to say one bounding box has more than one object detected if that is the case its over-fitting lower epochs keep eye on loss value anything lower than 0.05 is considered as good not in all cases, You can adjust threshold settings set it to higher confidence ratio >=80% you would see less no of boxes
